### ISSUE TITLE
Create series of examples with tutorials

### ISSUE THREAD

**Issue by chshersh** [kowainik/co-log#84]
> Currently all tutorials are here: https://github.com/kowainik/co-log/tree/master/co-log/example

Proposed by @vrom911. Basically, have `examples` directory with different complete usage examples of `co-log` library. I propose to have the following examples:

- [x] Basic example where you pass `LogAction IO String` as an argument and feed messages to it with `<&` operator
- [ ] Same example but with `LoggerT` monad and simple message
- [ ] Same example but with `LoggerT` monad and rich message
- [ ] Basic example with `LoggerT` but not printing `threadId`
- [ ] Like with `LoggerT` monad but using `withLog` function to change logger action locally
- [ ] Using `PureLogger`
- [x] Using custom monad with `LogAction` inside `Env`
- [ ] Logger combinators
- [ ] Logger rotation (#133)

Should be good for now :+1: Maybe there will be some code duplication between examples but I think that it's better to have all code in single module for the example to be able to see full code.

Not to say that each example should contain pretty good documentation :smirk: 

Things to do after finishing:

- [ ] Remove `play-co-log` executable
- [x] Rename `example` directory to `tutorials`
- [ ] Change README for `co-log`

**Comment by pangwa** [comment]
+1, is there any example on terminal logging and log rotation?

**Comment by chshersh** [comment]
@pangwa I agree that having an example of logging rotation would be good. I've added it to the list (and I see that you've also opened a separate issue and PR, will review in a moment). Regarding logging to terminal: currently, all examples are using logging to the terminal.
